### DISCUSSION TITLE
[4.0] UX Feature Request - Preview Module Positions - Too many clicks

### DISCUSSION THREAD

**Discussion by coolcat-creations** [joomla/joomla-cms#47169] [2. Cold Features]
### Is your feature request related to a problem? Please describe.

If you are new to Joomla and you don't know about the module-positions you have to do several clicks in the options in template Manager to activate Module Position Preview. 

For experienced Joomla users it's not a problem. Most of the experienced Joomla user would not need this option at all. So it would be great to make it easier for new users.

![grafik](https://user-images.githubusercontent.com/828371/80458971-f41a7980-8931-11ea-9f0a-b5f16cabb055.png)

### Describe the solution you'd like
It would be great to open a modal if one clicks on the crossed eye of a Template style saying:
"Module Position Preview is disabled by default for security reasons do you want to enable it now?" After clicking on yes the option could be activated and show an external Link Icon next to the eye?

That's at least easier to explain in Joomla training than describing all the clicks the user has to go through.

**Comment by brianteeman** [comment]
This was done in the proposal at https://github.com/joomla-projects/j4adminui

**Comment by ghost** [comment]
> It would be great to open a modal if one clicks on the crossed eye of a Template style saying:
> "Module Position Preview is disabled by default for security reasons do you want to enable it now?" After clicking on yes the option could be activated and show an external Link Icon next to the eye?

Agree.
